### PR TITLE
docs/managed-victoriametrics: update image size, add script to resize images

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Added make command, which helps to reduce image size by resizing their resolution.
How to use it:

1. From the docs folder, just run
```bash
PATH_TO_IMAGES=managed-victoriametrics IMAGES_EXTENSION=png make docs-change-images
```

Made resize of all images for `managed-victoriametrics` folder. The size was reduced from 29M to 2.5M.

Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -20,3 +20,8 @@ docs-up-docker:
 		-p 4000:4000 \
 		-v $(PWD):/srv/jekyll \
 		jekyll/jekyll:3.8 jekyll serve --livereload
+
+docs-images-resize:
+	docker run --rm -it \
+ 		-v $(PWD)/$(PATH_TO_IMAGES):/imgs \
+ 		dpokidov/imagemagick /imgs/*.$(EXTENTION) -set filename:f "%t" -resize 800x600 /imgs/%[filename:f].$(EXTENTION)
```
Comment (hagen1778) [inline docs/Makefile:27]: how do I set the correct PATH_TO_IMAGES and EXTENTION (pls note a typo in this word) variables correctly?

Why the resolution is 800x600? Does it respect aspect ratio? 

Should we try converting to webp instead?

File: docs/Makefile [hunk 2]
```diff
@@ -20,3 +20,27 @@ docs-up-docker:
...
+#
+# This command will reduce all images with extionsion and folder defined in the command.
+# All images with defined extensions will be removed.
+# After that it is nesessary to update src attribute for image tag
+
+docs-images-to-webp:
+	docker run -it --rm -v \
+		$(PWD)/$(PATH_TO_IMAGES):/data \
+		elswork/cwebp \
+		find ./ -type f -name '*.$(IMAGES_EXTENSION)' -exec sh -c 'cwebp -q 20 -m 6 $$1 -o "$${1%.png}.webp"' _ {} \;
```
Comment (hagen1778) [inline docs/Makefile:39]: quality 20 seems like too low, I clearly see image quality degradation. We need to remember that these images can be re-used by VM users. Would be pity if the images will have that low quality.

File: docs/Makefile [hunk 3]
```diff
@@ -20,3 +20,27 @@ docs-up-docker:
...
+
+# To run image convertor just run next command in the terminal
+#
+# PATH_TO_IMAGES=path_to_the_images IMAGES_EXTENSION=png make docs-change-images
+#
+# This command will reduce all images with extionsion and folder defined in the command.
+# All images with defined extensions will be removed.
+# After that it is nesessary to update src attribute for image tag
+
+docs-images-to-webp:
```
Comment (hagen1778) [inline docs/Makefile:35]: How do you see the procedure of adding a new doc and images for it? When should we call this command?

File: docs/Makefile [hunk 4]
```diff
@@ -20,3 +20,27 @@ docs-up-docker:
 		-p 4000:4000 \
 		-v $(PWD):/srv/jekyll \
 		jekyll/jekyll:3.8 jekyll serve --livereload
+
+# This commands is helpful when you want to convert image to the webp format
```
Comment (hagen1778) [inline docs/Makefile:24]: docs in the root folder, such as https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md, could have references to images in `docs` folder.

File: docs/Makefile (hunk 2 above)
Comment (dmitryk-dk) [reply docs/Makefile:39]: Hi @hagen1778 ! Can you show what images have degradation I will check what produce it degradation, or maybe recreate them

Comment (hagen1778) [reply docs/Makefile:39]: I've compared two following images:
* https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/9149353a364b3fae49a82d4740a1c3946e0c4569/docs/Cluster-VictoriaMetrics_cluster-scheme.png
* https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/e1eb0c5d06bb1c86efb0a7b3aa0be6b7542c767a/docs/Cluster-VictoriaMetrics_cluster-scheme.webp

I find text on images to have obscure artifacts.

Comment (dmitryk-dk) [reply docs/Makefile]: used quality 40% it looks like better

File: docs/Makefile (hunk 3 above)
Comment (dmitryk-dk) [reply docs/Makefile:24]: Described it in the README.md

File: docs/Makefile (hunk 4 above)
Comment (dmitryk-dk) [reply docs/Makefile]: Updated README.md

**Comment by hagen1778** [comment]
Hey @dmitryk-dk! Sorry for the late reply and asking for additional changes. I'm afraid converting all images to webp could be a source of consufion for docs from subpackages (vmalert/vmagent etc), docs synced from operator or datasource repos.
I still think this PR is great, but it should be applied to guides and managed VM sections - docs without auto-generation or sync capabilities. wdyt?

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Pull request merged**